api4/emoji.go

```diff
@@ -37,11 +37,6 @@ func createEmoji(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if len(*c.App.Config().FileSettings.DriverName) == 0 {
```

Comment:
I called this out in a [separate commit](https://github.com/mattermost/mattermost-server/pull/10243/commits/41cf64c5ef20d7c97c7147a843eab9787bb90a6a), but in short I moved these checks into the `App` layer as they are a config issue not a permission issue.

I couldn't "fix" the test because it's no longer possible to use `App.UpdateConfig()` to write an invalid config (followed by an `App.PersistConfig()` that would render the server unable to start next time!). In another place, I used the new `MemoryStore` which skips such checks to continue to allow the test, but I didn't want to pull this test apart too much to enable that here.